pkg/scheduler/cache/event_handlers.go

```diff
@@ -240,9 +240,16 @@ func (sc *SchedulerCache) addNode(node *v1.Node) error {
 }
 
 func isNodeInfoUpdated(oldNode, newNode *v1.Node) bool {
-	return !reflect.DeepEqual(oldNode.Status.Allocatable, newNode.Status.Allocatable) ||
-		!reflect.DeepEqual(oldNode.Spec.Taints, newNode.Spec.Taints) ||
-		!reflect.DeepEqual(oldNode.Labels, newNode.Labels)
+	switch {
+	// TODO(jiaxuanzhou): reserved for predictions for volumes.
+	case !reflect.DeepEqual(oldNode.Status, newNode.Status):
```

Comment:
Why not check `Allocatable` directly?

Reply:
For field of ```status``` for node, there are several potential predictions as ```disk presure, volumes check , mem presure. etc.``` when scheduling . So i list three logics for this check: ```status, spec, metaData of the node```, and ```Allocatable``` is part of the ```status```.

Reply:
BTW, for resource calculating of the ```func (ni *NodeInfo) AddTask(task *TaskInfo)```, there is not any mutex lock for nodeinfo, even the tasks(pods) were handled sequentially , but it is easy to happen that the ```nodeInfo``` for the next task(pod) is not the updated one, so the worst result is several pods were bind  to the same node and  the pods  will be reported  not enough  of cpu 、 mem or GPU..